### PR TITLE
Feature/homepage mobile ui updates

### PR DESCRIPTION
**Pull request by kedarguy**
**Related issue(s):** #0

## Description

**Notes:**

## Checklist

  * [ ] Approved by designer

Usability:
  * [ ] Developer
  * [ ] Project manager

Tests
  
  * [ ] UI snapshot testing 
  * [ ] Unit testing
  * [ ] Integration testing
  * [ ] E2E testing

### CODE REVIEW

File: packages/components/htz-components/src/components/List/views/Slugs/SlugsView.js [hunk 1]
```diff
@@ -95,6 +95,7 @@ export default function Slugs({
       areasTemplate={areasTemplate}
       innerBackgroundColor="transparent"
       rowTemplate="auto auto auto 1fr auto"
+      rowGap={[ { until: 's', value: '2rem', }, { from: 's', value: '1rem', }, ]}
```
Comment (TxHawks) [inline packages/components/htz-components/src/components/List/views/Slugs/SlugsView.js]: `<ListView />` has a default value for `gridGap`, so `rowGap` isn't guarantied to win over the default. You need to explicitly pas `gridGap={null}`. 

If you also need `colGap`, specify it separately.

File: packages/components/htz-components/src/components/List/views/Pazuzu/PazuzuView.js [hunk 2]
```diff
@@ -44,14 +44,12 @@ export default function Pazuzu({
     <ListView
       disableWrapper
       keepHeadingLevel
-      colGap="4"
+      colGap={[ { until: 's', value: '2', }, { from: 's', value: '4', }, ]}
       rowGap={null}
```
Comment (TxHawks) [inline packages/components/htz-components/src/components/List/views/Pazuzu/PazuzuView.js]: `rowGap` doesn't have a default. No need to set `null` explicitly.

```suggestion
```

File: packages/components/htz-components/src/components/List/views/Morbo/MorboView.js [hunk 3]
```diff
@@ -56,6 +56,7 @@ export default function Morbo({
         { from: 'l', until: 'xl', value: 'repeat(4, 1fr)', },
         { from: 'xl', value: 'repeat(6, 1fr)', },
       ]}
+      rowGap={[ { until: 's', value: '2rem', }, { from: 's', value: '1rem', }, ]}
```
Comment (TxHawks) [inline packages/components/htz-components/src/components/List/views/Morbo/MorboView.js]: `<ListView />` has a default value for `gridGap`, so `rowGap` isn't guarantied to win over the default. You need to explicitly pas `gridGap={null}`. 

If you also need `colGap`, specify it separately.

Comment (kedarguy) [reply packages/components/htz-components/src/components/List/views/Morbo/MorboView.js]: done

**Pull request merged**